bin/_blackbox_common.sh

```diff
@@ -277,6 +277,9 @@ function shred_file() {
     #NOTE: srm by default uses 35-pass Gutmann algorithm
     CMD=srm
     OPT=-f
+  elif _F=$(mktemp); rm -P "${_F}" >/dev/null 2>/dev/null ; then
```

Comment:
Should it be the `CMD` on the line below which is altered? This is deleting a temporary file that we use to test for the flag.